testnets/nilliontestnet/assetlist.json

```diff
@@ -0,0 +1,34 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "nilliontestnet",
+  "assets": [
+    {
+      "description": "NIL - the native token of Nillion",
+      "denom_units": [
+        {
+          "denom": "unil",
+          "exponent": 0,
+          "aliases": []
+        },
+        {
+          "denom": "nil",
+          "exponent": 6,
+          "aliases": []
+        }
+      ],
+      "base": "unil",
+      "name": "Nillion",
+      "display": "nil",
+      "symbol": "NIL",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/nilliontestnet/images/nil.png"
+      },
+      "coingecko_id": "",
```

Comment:
Please remove properties with empty values, like this coingekco_id

Reply:
Fixed